### PR TITLE
chore(website): update FAQ.md

### PR DESCRIPTION
**Pull request by archie0x**
I noticed that the FAQ on the EVM was a copy-paste from CoinMarketCap, so I thought I'd rewrite it to make it sound a bit more authentic. I've also added an FAQ on ZK-SNARKs. Also did a few other small edits. Hope you don't mind!

### CODE REVIEW

**Comment by umededoteth** [comment]
Let me know what you guys think!

File: packages/website/docs/intro/faq.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Taiko is a decentralized Ethereum-equivalent ZK-EVM and general-purpose ZK-Rollu
 
 Currently, only the Privacy and Scaling Explorations (PSE) team and Taiko are working on a Type 1 ZK-EVM. This means it aims to be Ethereum-equivalent.
 
-You can read more about the difference between a Type 1 ZK-EVM and others from our blog post, [The Type 1 ZK-EVM](https://mirror.xyz/labs.taiko.eth/w7NSKDeKfJoEy0p89I9feixKfdK-20JgWF9HZzxfeBo).
+You can read more about the difference between a Type 1 ZK-EVM and others from our blog post [The Type 1 ZK-EVM](https://mirror.xyz/labs.taiko.eth/w7NSKDeKfJoEy0p89I9feixKfdK-20JgWF9HZzxfeBo).
```
Comment (d1onys1us) [inline packages/website/docs/intro/faq.md:11]:
```suggestion
You can read more about the difference between a Type 1 ZK-EVM and others from our blog post: [The Type 1 ZK-EVM](https://mirror.xyz/labs.taiko.eth/w7NSKDeKfJoEy0p89I9feixKfdK-20JgWF9HZzxfeBo).
```

File: packages/website/docs/intro/faq.md [hunk 2]
```diff
@@ -37,37 +35,43 @@ Rollups conduct transactions on L2, which is quicker and allows for batching, an
...
-## What is an EVM?
+## What is the Ethereum Virtual Machine (EVM)?
 
-Developers utilize an Ethereum virtual machine, often known as a "virtual computer," as a software platform to build decentralized applications (dApps) and to run and deploy smart contracts on the Ethereum network. All Ethereum accounts and smart contracts exist on this virtual machine. The EVM's purpose is to determine what the overall state of Ethereum for each block in the blockchain will be.
+The Ethereum Virtual Machine (EVM) is a software platform that developers can use to deploy and run smart contracts on the Ethereum blockchain. Also known as a "virtual computer", the EVM enforces the rules of the Ethereum protocol and ensures the integrity of the Ethereum blockchain by processing and validating transactions. Developers can use the EVM to build decentralized applications (dApps) in a secure and trusted manner.
 
 [Learn more about the EVM](https://ethereum.org/en/developers/docs/evm/)
 
-## What is a ZK-EVM?
+## What is the Zero-Knowledge Ethereum Virtual Machine (ZK-EVM)?
```
Comment (d1onys1us) [inline packages/website/docs/intro/faq.md:52]:
```suggestion
## What is a Zero-Knowledge Ethereum Virtual Machine (ZK-EVM)?
```

File: packages/website/docs/intro/faq.md [hunk 3]
```diff
@@ -37,37 +35,43 @@ Rollups conduct transactions on L2, which is quicker and allows for batching, an
...
-## What are Zero-Knowledge Proofs (ZKPs)?
+## What is a Zero-Knowledge Proof (ZKP)?
 
 A zero-knowledge proof is a method by which one party (the prover) can prove to another party (the verifier) that something is true, without revealing any information apart from the fact that this specific statement is true.
 
 There are 2 types of zero-knowledge proofs: ZK-SNARKs and ZK-STARKs. Taiko uses ZK-SNARKs in its design.
 
+## What is a ZK-SNARK?
+
+A Zero-Knowledge Succinct Non-Interactive Argument of Knowledge (ZK-SNARK) is a type of zero-knowledge proof in which one party can demonstrate the possesion of certain information without revealing the information itself and having any direct communication between the party providing the proof and the party verifying it. ZK-SNARKs are known to be lighter than zkSTARKs and take less time to verify. They also require less gas, offering cheaper transactions.
```
Comment (d1onys1us) [inline packages/website/docs/intro/faq.md:66]:
```suggestion
A Zero-Knowledge Succinct Non-Interactive Argument of Knowledge (ZK-SNARK) is a type of zero-knowledge proof in which one party can demonstrate the possesion of certain information without revealing the information itself and having any direct communication between the party providing the proof and the party verifying it. ZK-SNARKs are known to be lighter than ZK-STARKs and take less time to verify. They also require less gas, offering cheaper transactions.
```

**Review by dionysuzx — COMMENTED** [review]
thank you for your contribution, I appreciate your attention to detail. I added a few suggestions, otherwise LGTM.

**Review by dantaik — APPROVED** [review]

**Review by dionysuzx — APPROVED** [review]

**Comment by umededoteth** [comment]
@d1onys1us @dantaik Thank you guys! I had some doubts about whether it's "the ZK-EVM" or "a ZK-EVM" but I see now where you're coming from Dave. 

Would be great to change the article here as well if we're sticking to the indefinite article. "The ZK-EVM proves the correctness...".

Thanks guys again!

**Pull request merged**